ClosedXML_Tests/Excel/CalcEngine/FunctionsTests.cs

```diff
@@ -658,5 +659,131 @@ public void TestOmittedParameters()
                 Assert.AreEqual(2, value);
             }
         }
+
+        [Test]
+        public void NumberValue()
+        {
+            Object actual;
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"123.45\", \".\", \",\")");
+            Assert.AreEqual(123.45, actual);
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"123,45\", \",\", \".\")");
+            Assert.AreEqual(123.45, actual);
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"123.456.789,01\", \",\", \".\")");
+            Assert.AreEqual(123456789.01, actual);
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"1,234.56\", \".\", \",\")");
+            Assert.AreEqual(1234.56, actual);
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"1.234,56\", \",\", \".\")");
+            Assert.AreEqual(1234.56, actual);
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"123.45\")");
+            Assert.AreEqual(123.45, actual);
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"1,234.56\")");
+            Assert.AreEqual(1234.56, actual);
+
+            actual = XLWorkbook.EvaluateExpr("NUMBERVALUE(\"+1\")");
```

Comment:
There were spaces In the edge cases I provided. And I believe they may be crucial.